ethpm/utils/manifest_validation.py

```diff
@@ -112,7 +112,7 @@ def validate_manifest_exists(manifest_id: str) -> None:
         )
 
 
-def format_manifest(manifest: Manifest, *, prettify: bool) -> str:
+def format_manifest(manifest: Manifest, *, prettify: Optional[bool]) -> str:
```

Comment:
I think you want to do `prettify: bool = None` which `mypy` will interpret as `Optional`.  Otherwise the function signature doesn't really make sense since at the python level you still are required to provide the argument *but* it is allowed to be `None` (which is weird)